README.md

```diff
@@ -98,6 +98,7 @@ Read about guest lectures [here](course/guest-lectures.md).
 * [Hsiao-Ching (Jean) Lin](people/hsiaoching-lin.md)
 * [Huai-Che (Acsa) Lu](people/huaiche-lu.md)
 * [Inna Shteinbuk](people/inna-shteinbuk.md)
+* [Jacqueline J.Pang](people/jacqueline-pang.md)
```

Comment:
Can I nitpick? There's a space after J. in the file :) It's ok, I'll let it slide.